daphne/src/messages.rs

```diff
@@ -443,17 +442,6 @@ impl Interval {
     pub fn end(&self) -> Time {
         self.start + self.duration
     }
-
```

Comment:
Reviewer note: This check is now implemented here: https://github.com/cloudflare/daphne/pull/131/files#diff-6edcf4a9c3adc1ac1e8061cd876a931a9dedf381bdf3ce49d70b34b95a697205R771-R777